src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/Cas1SiteSurveyPremiseFactory.kt

```diff
@@ -0,0 +1,160 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.seed.cas1
+
+import org.jetbrains.kotlinx.dataframe.AnyCol
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.getColumn
+import org.jetbrains.kotlinx.dataframe.name
+
+class Cas1SiteSurveyPremiseFactory {
```

Comment:
We should be using factories for the room characteristic job too..

Reply:
Yep, it helps split the logic up. Also this should make it easy for you to grab the qcode in that job from the first worksheet, with a little bit of refactoring